src/docs.js

```diff
@@ -0,0 +1,179 @@
+  },
+  {
+    name: "Starknet",
+    link: "https://docs.apeworx.io/ape-starknet/stable/",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Polygon",
+    link: "https://github.com/ApeWorX/ape-polygon/",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Polygon zkEVM",
+    link: "https://github.com/ApeWorX/ape-polygon-zkevm/",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Base",
+    link: "https://github.com/ApeWorX/ape-base/",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Fantom",
+    link: "https://github.com/ApeWorX/ape-fantom/",
+    section: sections.PLUGIN
+  },
+  {
+    name: "BSC (Binance Smart Chain)",
+    link: "https://github.com/ApeWorX/ape-bsc/",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Optimism",
+    link: "https://github.com/ApeWorX/ape-optimism",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Arbitrum",
+    link: "https://github.com/ApeWorX/ape-arbitrum",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Avalanche",
+    link: "https://github.com/ApeWorX/ape-avalanche",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Flashbots",
+    link: "https://github.com/ApeWorX/ape-flashbots",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Ledger",
+    link: "https://github.com/ApeWorX/ape-ledger",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Trezor",
+    link: "https://github.com/ApeWorX/ape-trezor",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Frame",
+    link: "https://github.com/ApeWorX/ape-frame",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Solidity",
+    link: "https://github.com/ApeWorX/ape-solidity",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Cairo",
+    link: "https://github.com/ApeWorX/ape-cairo",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Foundry",
+    link: "https://github.com/ApeWorX/ape-foundry",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Hardhat",
+    link: "https://github.com/ApeWorX/ape-hardhat",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Ganache",
+    link: "https://github.com/ApeWorX/ape-ganache",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Jupyter Notebook",
+    link: "https://github.com/ApeWorX/ape-notebook",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Tokens",
+    link: "https://github.com/ApeWorX/ape-tokens",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Titanoboa",
+    link: "https://github.com/ApeWorX/ape-titanoboa",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Address Book",
+    link: "https://github.com/ApeWorX/ape-addressbook",
+    section: sections.PLUGIN
+  },
+  {
+    name: "AWS KMS",
+    link: "https://github.com/ApeWorX/ape-aws-kms",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Chainstack",
+    link: "https://github.com/ApeWorX/ape-chainstack",
+    section: sections.PLUGIN
+  },
+  {
+    name: "Plugin Template",
+    link: "https://github.com/ApeWorX/ape-template",
+    section: sections.PLUGIN
+  },
+  {
+    name: "EIP712",
```

Comment:
There are a couple of others under other here
- evm-trace
- ethpm-types
- eth-pydantic-types